deepchem/models/tensorgraph/optimizers.py

```diff
@@ -101,6 +101,29 @@ def _create_optimizer(self, global_step):
     return tf.train.GradientDescentOptimizer(learning_rate=learning_rate)
 
 
+class PowerSignOptimizer(Optimizer):
+  """The new powersign optimization algorithm"""
```

Comment:
For consistency with the other optimizers, we should just call the class `PowerSign`, not `PowerSignOptimizer`.  And you probably don't want to refer to the algorithm as "new", because a few years from now it won't be, but the docstring will still be the same. :)